package.json

```diff
@@ -4,17 +4,19 @@
   "description": "React component library for the Bare Metal Installer",
   "license": "Apache-2.0",
   "repository": "openshift-assisted/assisted-ui-lib",
-  "main": "dist/index.js",
-  "module": "dist/index.modern.js",
-  "source": "src/index.ts",
-  "types": "dist/src/index.d.ts",
+  "main": "dist/src/index.js",
   "engines": {
-    "node": ">=10"
+    "node": ">=12"
+  },
+  "exports": {
+    ".": "./dist/src/index.js"
   },
   "scripts": {
     "prebuild": "run-s test",
-    "build": "microbundle-crl --no-compress --format modern,cjs --css-modules false",
-    "start": "microbundle-crl watch --no-compress --format modern,cjs --css-modules false",
+    "build": "tsc && yarn copy",
+    "start": "tsc --watch & yarn watch-css",
+    "copy": "copyfiles \"./src/**/*.css\" \"./dist/\"",
+    "watch-css": "nodemon --watch ./src -e css --exec \"yarn copy\"",
     "start-assisted-ui": "./scripts/start-assisted-ui.sh",
     "start-uhc-portal": "./scripts/start-uhc-portal.sh",
     "sync-to-ui": "./scripts/sync-dist.sh",
```

Comment:
@rawagner do we still need this script (`sync-to-ui`)? if not we can remove it and also update the README since there's no need to install `inotify` anymore  
How would you launch now the dev environment?

Right now the `start-assisted-ui` script does this: `yarn start" "yarn sync-to-ui" "cd ../assisted-ui; yarn start`
IIUC after this PR it should be only about executing `yarn start` here and `yarn start` in assisted-ui, am I right?

Reply:
We still need it. The purpose of `sync-to-ui` is syncing the `dist` folder to standalone, uhc or any other project. 

This PR changes only the content of dist folder (its not bundled, but transpiled only), its not doing any changes to how we sync the dist to consumer app.

There is no change when running the dev env.
You will still run `yarn start-assisted-ui` or `yarn start-uhc-portal`